.github/workflows/pull_requests.yaml

```diff
@@ -36,6 +36,12 @@ jobs:
   Vulnerability-Scanning:
     name: vulnerability-scanning
     runs-on: ubuntu-latest
+    env:
+      # USE WITH CAUTION, IT IMPLIES RISK
+      # This is a list of vulnerabilities that will be skipped from the check
+      # It is mostly used for vulnerabilities which haven't fixed by the depending projects
+      # The format is space separated string: "CVE-XXXX-XXXX CVE-XXXX-XXXX CVE-XXXX-XXXX"
+      SKIPPED_VULN: ""
```

Comment:
can we comment the whole `env` section?

Reply:
I could comment it, but if I do that then the line 56 will fail, as SKIPPED_VULN is not defined.
Therefore I would be forced to comment that whole step, which would be equivalent to not submitting anything in the end :-)